sdk/containerregistry/azure-containerregistry/samples/sample_upload_download_manifest.py

```diff
@@ -72,13 +72,28 @@ def upload_download_manifest(self):
             downloaded_manifest = download_manifest_result.manifest
             download_manifest_stream = download_manifest_result.data
             print(b"".join(download_manifest_stream))
-            # Download the layers
+            # Download and write out the layers
             for layer in downloaded_manifest.layers:
-                with client.download_blob(repository_name, layer.digest).data as layer_stream:
-                    print(b"".join(layer_stream))
-            # Download the config
-            with client.download_blob(repository_name, downloaded_manifest.config.digest).data as config_stream:
-                print(b"".join(config_stream))
+                # Remove the "sha256:" prefix from digest
+                layer_file_name = layer.digest.split(":")[1]
+                try:
+                    with client.download_blob(repository_name, layer.digest) as stream:
```

Comment:
Can we add the async sample?

Reply:
I was planning to add it in manifest PR in which the async manifest operations will be available. If we add async sample, it will only include `async blob upload and download` samples.